src/read.c

```diff
@@ -4292,7 +4301,7 @@ static avifResult avifMetaFindAlphaItem(avifMeta * meta,
 }
 
 #if defined(AVIF_ENABLE_EXPERIMENTAL_GAIN_MAP)
-// Finds a 'tmap' (tone mapped image item) box associated with the given 'colorItem'.
+// Finds a 'tmap' (tone mapped image item) box associated with the given color item.
```

Comment:
Optional: Undo this change (i.e., change `color item` back to `'colorItem'`) ?

Reply:
Done.